ui/src/timeMachine/selectors/index.test.ts

```diff
@@ -104,16 +104,17 @@ describe('TimeMachine.Selectors.Index', () => {
   })
   it(`getEndTime should return ${newYears} when lower is ${date}`, () => {
     const timeRange = {
-      lower: 'now() - 30d',
+      lower: date,
```

Comment:
This was updated since it was an unrealistic input to expect the lower to be set to a timeRange dropdown selection with the upper property being set to a date.